src/js/components/sharedComponents/checkbox/AccordionCheckboxSecondary.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * AccordionCheckboxSecondary.jsx
+ * Created by Josue Aguilar on 09/11/2024.
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+
+const propTypes = {
+    category: PropTypes.object,
+    selectedFilters: PropTypes.object,
+    singleFilterChange: PropTypes.func,
+    filters: PropTypes.object,
+    expanded: PropTypes.bool
+};
+
+// sub-filters hidden from the user, but  passed to the API when the parent filter is selected
+const excludedSubFilters = "IDV_B";
+
+const AccordionCheckboxSecondary = ({
+    category, selectedFilters, singleFilterChange, filters, expanded
+}) => {
+    const selectFilter = (filter) => {
+        const selection = {
+            value: filter
+        };
+        console.log(filter);
```

Comment:
remove console

Reply:
This isn't a feature that's being released so I think it's fine for now.